docs/core-concepts.md

```diff
@@ -106,7 +106,6 @@ The main function of React-RxJS, `bind`, uses this operator on every stream. `bi
 - Leveraging Error Boundaries to allow graceful error recovery.
 - Performance optimizations, making sure React doesn't update when it doesn't need to.
 - Manages a cache of parametric observables (when using the factory overload).
-- Delays unsubscriptions from hooks to keep the stream alive between re-renders.
```

Comment:
I'm not sure on removing this point...

Reply:
I think that it's best to remove it... At least for now, it's a bit of an implementation detail, something that we are doing to prevent unnecessary re-subscriptions. However, it's not something that the consumer should rely on. So, I think that it's wise to remove it.